### DISCUSSION TITLE
Everyday Usage macOS install translate-shell,  doesn't work...

### DISCUSSION THREAD

**Discussion by cocoyamm** [Homebrew/discussions#4351] [Everyday usage]
### Output of `brew config`

```shell
Homebrew 4.0.9

translate-shell 0.9.7.1
```

### Output of `brew doctor`

```shell
When I try to writing "trans[word]" doesn't work....
```

### Description of issue

When I try to writing "trans[word]" doesn't work....
![IMG_6169](https://user-images.githubusercontent.com/122199469/227124421-bd2e6d89-cc1f-4e07-b712-b74a2295101f.JPEG)
![IMG_6168](https://user-images.githubusercontent.com/122199469/227124420-232fef3f-1545-4f60-9ed6-3e0b8ff9523d.JPEG)
<img width="533" alt="截屏2023-03-23 14 45 40" src="https://user-images.githubusercontent.com/122199469/227124535-22a111c8-8522-487d-96af-6677d3b64260.png">

**Comment by gromgit** [comment]
Please post text output _as text_, not as screenshots. Two of the three screenshots you posted are empty; posting text _as text_ never fails like this.
